verrazzano/definer.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package verrazzano
+
+import (
+	"bytes"
+	"io"
+)
+
+// This class is typically wrapped around a TypeFilteringReadCloser, which
```

Comment:
did we write this? it looks suspicious.  if we copied it from somewhere, it needs an attribution

Reply:
yes, I confirmed that we wrote this